agreement_legal/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2018 - TODAY, Pavlov Media
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Agreements Legal",
+    "summary": "Manage Agreements, LOI and Contracts",
+    "author": "Pavlov Media, "
+    "Open Source Integrators, "
+    "Yves Goldberg (Ygol Internetwork), "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
+    "category": "Partner",
+    "license": "AGPL-3",
+    "version": "12.0.1.0.0",
+    "depends": ["contacts", "agreement", "mail", "product"],
```

Comment:
@ygol No need for `mail` module in dependency because it is already in the `contacts` module.